src/main/kotlin/no/fdk/fdk_event_harvester/controller/UpdateController.kt

```diff
@@ -0,0 +1,27 @@
+package no.fdk.fdk_event_harvester.controller
+
+import no.fdk.fdk_event_harvester.service.EndpointPermissions
+import no.fdk.fdk_event_harvester.service.UpdateService
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import javax.servlet.http.HttpServletRequest
+
+@RestController
+@RequestMapping(value = ["/update"])
+class UpdateController(
+    private val endpointPermissions: EndpointPermissions,
+    private val updateService: UpdateService) {
+
+    @PostMapping("/meta")
+    fun updateMetaData(httpServletRequest: HttpServletRequest): ResponseEntity<Void> =
```

Comment:
Ekstraher direkte verdi som trengs fra request, f.eks header vha `@RequestHeader`

Reply:
Hvorfor bruker vi ikke Spring Security?

Reply:
Hele api-key greia vi kjører med kom ut fra noe vi gjorde veldig kjapt for et annet api i høst, eneste måten jeg fant for å støtte det i spring-security da var å skrive et nytt filter selv. Har fortsatt ikke prioritert å sette opp et slikt filter...

Reply:
Men når dere påpeker det, burde egentlig ikke bruke api-key funksjonaliteten her. Det er bare når vi trenger at andre api skal bruke endepunktet, men dette endepunktet er det strengt tatt bare @valosnah som skal bruke. Burde sette opp token-støtte i stedet.

Reply:
Lar det være api-key enn så lenge, lagde issue for å gå over til token: https://github.com/Informasjonsforvaltning/fdk-event-harvester/issues/3